libraries/AP_HAL_Linux/RCInput_Aero.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2017  Intel Corporation. All rights reserved.
+ *
+ * This file is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This file is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+ 
+/* feature added by Anemos Technologies : Philippe Crochat (pcrochat@anemos-technologies.com) */
+
+#pragma once
+
+#define DEVICE_NAME "aeroio"
+#define CHANNEL_NUMBER 8
+
+#define CHANNEL_MIN_VALUE 970
+#define CHANNEL_MAX_VALUE 2000
+#define MAX_DIFF_VALUE 300
+#define MAX_RETRY 10
+#define CLOCK_REG 0x4a
+#define CHANNEL_REG 0x4b
+#define COMPLEMENTARY_FILTER 0.93
```

Comment:
all these defines should be in the .cpp, not the header. Reason is that any other part of the code that might want to use a MAX_RETRY constant will get its constant redefined in the moment it includes this header, so it's good to encapsulate the code as much as possible.

Reply:
ok

Reply:
done